lightningd/plugin_hook.c

```diff
@@ -401,25 +402,8 @@ void plugin_hook_add_deps(struct plugin_hook *hook,
 	add_deps(&h->after, buffer, after);
 }
 
-/* From https://en.wikipedia.org/wiki/Topological_sorting#Kahn's_algorithm
- * (https://creativecommons.org/licenses/by-sa/3.0/):
- * L ← Empty list that will contain the sorted elements
- * S ← Set of all nodes with no incoming edge
- *
- * while S is not empty do
- *     remove a node n from S
- *     add n to L
- *     for each node m with an edge e from n to m do
- *         remove edge e from the graph
- *         if m has no other incoming edges then
- *             insert m into S
- *
- * if graph has edges then
- *     return error   (graph has at least one cycle)
- * else
- *     return L   (a topologically sorted order)
- */
 struct hook_node {
+	bool finished;
```

Comment:
What are the semantics of `finished`? It's only used in sorting so I'd assume it means that it is in its final position.

Reply:
Included in sorted results. I'll add a comment